kube_apiserver_metrics/metadata.csv

```diff
@@ -58,4 +58,10 @@ kube_apiserver.grpc_client_started_total,count,,request,,The total number of RPC
 kube_apiserver.requested_deprecated_apis,gauge,,request,,"Gauge of deprecated APIs that have been requested, broken out by API group, version, resource, subresource, and removed_release",0,kubernetes_api_server_metrics,deprecated API requests,
 kube_apiserver.apiserver_admission_webhook_fail_open_count.count,count,,,,"Admission webhook fail open count, identified by name and broken out for each admission type (validating or mutating).",0,kubernetes_api_server_metrics,webhook fail open count,
 kube_apiserver.apiserver_admission_webhook_fail_open_count,gauge,,,,"Admission webhook fail open count, identified by name and broken out for each admission type (validating or mutating).",0,kubernetes_api_server_metrics,webhook fail open count,
+kube_apiserver.apiserver_envelope_encryption_dek_cache_fill_percent,gauge,,,,"Percent of the cache slots currently occupied by cached DEKs.",0,kubernetes_api_server_metrics,envelope encryption dek cache fillpercent,
+kube_apiserver.aggregator_unavailable_apiservice,gauge,,,,"Gauge of APIServices which are marked as unavailable broken down by APIService name.",0,kubernetes_api_server_metrics,aggregator unavailable apiservice,
+kube_apiserver.apiserver_flowcontrol_current_executing_requests,gauge,,,,"Number of requests in initial (for a WATCH) or any (for a non-WATCH) execution stage in the API Priority and Fairness subsystem",0,kubernetes_api_server_metrics,flowcontrol request concurrency limit,
+kube_apiserver.apiserver_flowcontrol_request_concurrency_limit,gauge,,,,"Shared concurrency limit in the API Priority and Fairness subsystem",0,kubernetes_api_server_metrics, flowcontrol request concurrency limit,
+kube_apiserver.apiserver_flowcontrol_rejected_requests_total.count,count,,,,"Number of requests rejected by API Priority and Fairness subsystem",0,kubernetes_api_server_metrics, flowcontrol rejected requests total,
+kube_apiserver.kubernetes_feature_enabled,gauge,,,,"Whether a Kubernetes feature gate is enabled or not, identified by name and stage (alpha; Kubernetes 1.26+)",0,kubernetes_api_server_metrics,kubernetes feature enabled,
```

Comment:
could you please remove this metric from the pull request because this metric was already added by https://github.com/DataDog/integrations-core/pull/14147 (and will be released in agent 7.45.0)

Reply:
I have synced with the master branch and removed the one which were added recently. Test case is failing with different error. Can you please take a look and let me know?